Dockerfile

```diff
@@ -0,0 +1,85 @@
+# https://www.caktusgroup.com/blog/2017/03/14/production-ready-dockerfile-your-python-django-app/
+# https://pythonspeed.com/articles/base-image-python-docker-images/ (July 2019)
+# FIXME: update for Python 3
+FROM python:2.7.16-slim-buster
```

Comment:
Might drop the minor version, at least when we upgrade to Python 3, so we get those updates with each rebuild.